doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/SiteRenderingContext.java

```diff
@@ -271,7 +290,7 @@ public void addSiteDirectory(File siteDirectory) {
      *
      * @return List of site directories files.
      */
-    public List<File> getSiteDirectories() {
+    public List<SiteDirectory> getSiteDirectories() {
```

Comment:
Please leave the old method in place, deprecate it and rename the new one (cannot come up with a good naming proposal right now). But please don't just break backwards compatibility.

Reply:
I cannot come up with a better one either, but am willing to take the BC break on my shoulders given that the scope of usage is quite limited.

Reply:
yes, Doxia Sitetools is only used by Maven Site plugin: binary compatibility is not really a topic, particularly when we are working on 2.0 milestones of Doxia Sitetools